LPS-171140 [FE] Allow multiple provider configurations

thektan opened the pull request:
https://issues.liferay.com/browse/LPS-171140

Follow-up to https://github.com/liferay/liferay-portal/pull/5181 with frontend updates to the react component.

Example format of how the form input for `textEmbeddingProviderConfigurations` is structured:
```json
[
    {
        "attributes": {
            "assetEntryClassNames": [
                "com.liferay.blogs.model.BlogsEntry",
                "com.liferay.journal.model.JournalArticle",
                "com.liferay.knowledge.base.model.KBArticle",
                "com.liferay.wiki.model.WikiPage"
            ],
            "languageIds": [
                "en_US"
            ],
            "maxCharacterCount": 500,
            "modelTimeout": 25,
            "model": "albert-base-v1",
            "huggingFaceAccessToken": "ok"
        },
        "cacheTimeout": 604800,
        "embeddingVectorDimensions": 768,
        "providerName": "huggingFaceInferenceAPI"
    }
]
```

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**search**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>search</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7873/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7873/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-32.liferay.com/job/test-portal-source-format/5403/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/5403/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 4 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ffaf63c35bea589c743101c834914c4829f43ef5">ffaf63c35bea589c743101c834914c4829f43ef5</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/thektan/liferay-portal/tree/LPS-171139_AllowMultipleProviderConfigurations">LPS-171139_AllowMultipleProviderConfigurations</a><br/>Branch GIT ID: <a href="https://github.com/thektan/liferay-portal/commit/c2f77cd3c71b3094e00f48f3a5406884b523f8d4">c2f77cd3c71b3094e00f48f3a5406884b523f8d4</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-32.liferay.com/job/test-portal-source-format/5403/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/5403/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-32.liferay.com/job/test-portal-source-format/5403/">test-portal-source-format#5403</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-32.liferay.com/userContent/jobs/test-portal-source-format/builds/5403/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-search/liferay-portal/pull/728">liferay-search#728</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2155988016">[master] ci:test:sf - thektan &gt; liferay-search - PR#728 - 2023-01-10[19:58:41]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-32.liferay.com/job/publish-testray-report/19435/">publish-testray-report#19435</a><br/></div></div>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:search - 135 out of 155 jobs passed in 1 hour 26 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ffaf63c35bea589c743101c834914c4829f43ef5">ffaf63c35bea589c743101c834914c4829f43ef5</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/ffaf63c35bea589c743101c834914c4829f43ef5">ffaf63c35bea589c743101c834914c4829f43ef5</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2154766853">EE Development Acceptance (master) - 4179 - 2023-01-10[05:57:32]</a></p><details><summary><strong>ci:test:search - 135 out of 155 jobs PASSED</strong></summary><div><h4>20 Failed Jobs:</h4><ul><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7873/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391707/">central-requirements-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369088/">functional-tomcat90-mysql57-jdk11_open/0/0</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366243/">functional-tomcat90-mysql57-jdk11_open/0/1</a></li></ul></li><li>functional-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394580/">functional-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348780/">functional-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391700/">functional-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348781/">functional-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369092/">functional-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337761/">functional-tomcat90-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366246/">functional-tomcat90-mysql57-jdk8/0/6</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366247/">functional-tomcat90-mysql57-jdk8/0/7</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327244/">functional-tomcat90-mysql57-jdk8/0/8</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174067/">functional-tomcat90-mysql57-jdk8/0/9</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376264/">functional-tomcat90-mysql57-jdk8/0/10</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337755/">functional-tomcat90-mysql57-jdk8/1/0</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394576/">functional-tomcat90-mysql57-jdk8/1/1</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394578/">functional-tomcat90-mysql57-jdk8/1/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348777/">functional-tomcat90-mysql57-jdk8/1/3</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369089/"><strike>functional-tomcat90-mysql57-jdk8/1/4</strike></a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376265/">functional-tomcat90-mysql57-jdk8/1/5</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348778/">functional-tomcat90-mysql57-jdk8/1/6</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369090/">functional-tomcat90-mysql57-jdk8/1/7</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376266/">functional-tomcat90-mysql57-jdk8/1/8</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337756/">functional-tomcat90-mysql57-jdk8/1/9</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348783/">functional-tomcat90-mysql57-jdk8/1/10</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376273/"><strike>functional-tomcat90-mysql57-jdk8/1/11</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312499/"><strike>functional-tomcat90-mysql57-jdk8/1/12</strike></a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327247/"><strike>functional-tomcat90-mysql57-jdk8/1/13</strike></a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337763/">functional-tomcat90-mysql57-jdk8/1/14</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394585/">functional-tomcat90-mysql57-jdk8/1/15</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415319/">functional-tomcat90-mysql57-jdk8/1/16</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174068/">functional-tomcat90-mysql57-jdk8/1/17</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339623/">functional-tomcat90-mysql57-jdk8/1/18</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224029/">functional-tomcat90-mysql57-jdk8/1/19</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224030/">functional-tomcat90-mysql57-jdk8/1/20</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376271/">functional-tomcat90-mysql57-jdk8/1/21</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312498/">functional-tomcat90-mysql57-jdk8/1/22</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366249/">functional-tomcat90-mysql57-jdk8/1/23</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415317/">functional-tomcat90-mysql57-jdk8/1/24</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327250/">functional-tomcat90-mysql57-jdk8/2/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369098/">functional-tomcat90-mysql57-jdk8/2/1</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339629/">functional-tomcat90-mysql57-jdk8/2/2</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174074/">functional-tomcat90-mysql57-jdk8/2/3</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415321/">functional-tomcat90-mysql57-jdk8/2/4</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369100/">functional-tomcat90-mysql57-jdk8/2/5</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337766/">functional-tomcat90-mysql57-jdk8/2/6</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391706/">functional-tomcat90-mysql57-jdk8/2/7</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369099/"><strike>functional-tomcat90-mysql57-jdk8/2/8</strike></a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369087/">functional-tomcat90-mysql57-jdk8/2/9</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376268/">functional-tomcat90-mysql57-jdk8/2/10</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339621/">functional-tomcat90-mysql57-jdk8/2/11</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337759/">functional-tomcat90-mysql57-jdk8/2/12</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391699/">functional-tomcat90-mysql57-jdk8/2/13</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366245/">functional-tomcat90-mysql57-jdk8/2/14</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415313/">functional-tomcat90-mysql57-jdk8/2/15</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337760/">functional-tomcat90-mysql57-jdk8/2/16</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369093/">functional-tomcat90-mysql57-jdk8/2/17</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348776/">functional-tomcat90-mysql57-jdk8/2/18</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366242/">functional-tomcat90-mysql57-jdk8/2/19</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394577/">functional-tomcat90-mysql57-jdk8/2/20</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366244/">functional-tomcat90-mysql57-jdk8/2/21</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337757/">functional-tomcat90-mysql57-jdk8/2/22</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224026/">functional-tomcat90-mysql57-jdk8/2/23</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174064/">functional-tomcat90-mysql57-jdk8/2/24</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348784/">functional-tomcat90-mysql57-jdk8/3/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391702/">functional-tomcat90-mysql57-jdk8/3/1</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224031/">functional-tomcat90-mysql57-jdk8/3/2</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174070/">functional-tomcat90-mysql57-jdk8/3/3</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394584/">functional-tomcat90-mysql57-jdk8/3/4</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366250/">functional-tomcat90-mysql57-jdk8/3/5</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327248/">functional-tomcat90-mysql57-jdk8/3/6</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312500/">functional-tomcat90-mysql57-jdk8/3/7</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391704/"><strike>functional-tomcat90-mysql57-jdk8/3/8</strike></a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369096/">functional-tomcat90-mysql57-jdk8/3/9</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337767/">functional-tomcat90-mysql57-jdk8/3/10</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/151314/">functional-tomcat90-mysql57-jdk8/3/11</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/146695/"><strike>functional-tomcat90-mysql57-jdk8/3/12</strike></a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348787/">functional-tomcat90-mysql57-jdk8/3/13</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/149010/">functional-tomcat90-mysql57-jdk8/3/14</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391708/">functional-tomcat90-mysql57-jdk8/3/15</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369101/">functional-tomcat90-mysql57-jdk8/3/16</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/147655/">functional-tomcat90-mysql57-jdk8/3/17</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312503/">functional-tomcat90-mysql57-jdk8/3/18</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174072/"><strike>functional-tomcat90-mysql57-jdk8/3/19</strike></a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348785/">functional-tomcat90-mysql57-jdk8/3/20</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366184/">functional-tomcat90-mysql57-jdk8/3/21</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327249/">functional-tomcat90-mysql57-jdk8/3/22</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376274/">functional-tomcat90-mysql57-jdk8/3/23</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366252/">functional-tomcat90-mysql57-jdk8/3/24</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391701/">functional-tomcat90-mysql57-jdk8/4/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339622/">functional-tomcat90-mysql57-jdk8/4/1</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415314/"><strike>functional-tomcat90-mysql57-jdk8/4/2</strike></a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415315/"><strike>functional-tomcat90-mysql57-jdk8/4/3</strike></a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376269/"><strike>functional-tomcat90-mysql57-jdk8/4/4</strike></a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224028/"><strike>functional-tomcat90-mysql57-jdk8/4/5</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394581/"><strike>functional-tomcat90-mysql57-jdk8/4/6</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174069/">functional-tomcat90-mysql57-jdk8/4/7</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369091/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348779/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174066/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415312/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415311/">functional-upgrade-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337758/">functional-upgrade-tomcat90-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339620/">functional-upgrade-tomcat90-mysql57-jdk8/0/6</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224027/">functional-upgrade-tomcat90-mysql57-jdk8/0/7</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376267/">functional-upgrade-tomcat90-mysql57-jdk8/0/8</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174065/">functional-upgrade-tomcat90-mysql57-jdk8/0/9</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337762/">functional-upgrade-tomcat90-mysql57-jdk8/0/10</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366182/">functional-upgrade-tomcat90-mysql57-jdk8/0/11</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376272/">functional-upgrade-tomcat90-mysql57-jdk8/0/12</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339625/">functional-upgrade-tomcat90-mysql57-jdk8/0/13</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369094/">functional-upgrade-tomcat90-mysql57-jdk8/0/14</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391703/">functional-upgrade-tomcat90-mysql57-jdk8/0/15</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366183/">functional-upgrade-tomcat90-mysql57-jdk8/0/16</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339626/">functional-upgrade-tomcat90-mysql57-jdk8/0/17</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369095/">functional-upgrade-tomcat90-mysql57-jdk8/0/18</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415318/">functional-upgrade-tomcat90-mysql57-jdk8/0/19</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394582/">functional-upgrade-tomcat90-mysql57-jdk8/0/20</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415316/">functional-upgrade-tomcat90-mysql57-jdk8/0/21</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376270/">functional-upgrade-tomcat90-mysql57-jdk8/0/22</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366248/">functional-upgrade-tomcat90-mysql57-jdk8/0/23</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327245/">functional-upgrade-tomcat90-mysql57-jdk8/0/24</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394583/">functional-upgrade-tomcat90-mysql57-jdk8/1/0</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348782/">functional-upgrade-tomcat90-mysql57-jdk8/1/1</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/327246/">functional-upgrade-tomcat90-mysql57-jdk8/1/2</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339624/">functional-upgrade-tomcat90-mysql57-jdk8/1/3</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224025/">js-unit-jdk8/0/0</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348788/">modules-integration-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337768/">modules-integration-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394588/"><strike>modules-integration-mysql57-jdk8/0/2</strike></a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366186/"><strike>modules-integration-mysql57-jdk8/0/3</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394587/">modules-integration-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312502/">modules-integration-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376275/"><strike>modules-integration-mysql57-jdk8/0/6</strike></a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348786/">modules-integration-mysql57-jdk8/0/7</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366253/">modules-integration-mysql57-jdk8/0/8</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337765/">modules-integration-mysql57-jdk8/0/9</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174071/"><strike>modules-integration-mysql57-jdk8/0/10</strike></a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224032/"><strike>modules-integration-mysql57-jdk8/0/11</strike></a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224033/">modules-semantic-versioning-jdk8/0/0</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339627/">modules-semantic-versioning-jdk8/0/1</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366185/">modules-semantic-versioning-jdk8/0/2</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369097/">modules-semantic-versioning-jdk8/0/3</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/339628/">modules-semantic-versioning-jdk8/0/4</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/174073/"><strike>modules-semantic-versioning-jdk8/0/5</strike></a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/415320/">modules-semantic-versioning-jdk8/0/6</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394586/">modules-semantic-versioning-jdk8/0/7</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/337764/">modules-semantic-versioning-jdk8/0/8</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312501/">modules-semantic-versioning-jdk8/0/9</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/348775/">modules-semantic-versioning-jdk8/0/10</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/224034/">modules-unit-jdk8/0/0</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/391705/">modules-unit-jdk8/0/1</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/366251/">semantic-versioning-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394579/">unit-jdk8/0/0</a></li></ul></li></ul></div><details><summary><strong>135 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7873/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376269//consoleText">functional-tomcat90-mysql57-jdk8/4/4</a><pre>     [exec] * What went wrong:
     [exec] Execution failed for task ':runPoshi'.
     [exec] &gt; There were failing tests. See the report at: file:///opt/dev/projects/github/liferay-portal/portal-web/tests/index.html
     [exec] 
     [exec] * Try:
     [exec] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Get more help at https://help.gradle.org
     [exec] 
     [exec] BUILD FAILED in 4m 56s
     [exec] Result: 1
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/376269
     [copy] Copying 4 files to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/376269/screenshots
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null7998717.properties

clean-up-selenium-driver:
     [echo] Executing commands: 
     [echo] killall chromedriver
     [exec] chromedriver: no process found
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null7998717.properties

clean-up-logs:
     [echo] stop.app.server.0.start.timestamp: 01-10-2023 20:43:52:737 PST
[beanshell] 2023-01-11 04:44:05.133 INFO  [main][ThemeHotDeployListener:171] 1 theme for speedwell-theme was unregistered
[beanshell] 2023-01-11 04:44:05.778 INFO  [main][ThemeHotDeployListener:171] 1 theme for classic-theme was unregistered
[beanshell] 2023-01-11 04:44:06.311 INFO  [main][ThemeHotDeployListener:171] 1 theme for minium-theme was unregistered
[beanshell] 2023-01-11 04:44:06.537 INFO  [main][ThemeHotDeployListener:171] 1 theme for dialect-theme was unregistered
[beanshell] 2023-01-11 04:44:06.839 INFO  [main][ThemeHotDeployListener:171] 1 theme for admin-theme was unregistered
[stopwatch] [stop.app.server.0: 19.055 sec]
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null7998717.properties

run-poshi-evaluate-console:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1786710928.properties

prepare-poshi-runner-properties:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/portal-web/poshi-ext.properties
[propertyfile] Updating property file: /opt/dev/projects/github/liferay-portal/.gradle/gradle.properties
     [echo] Executing Gradle task: evaluatePoshiConsole</pre></div></li><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/394581//consoleText">functional-tomcat90-mysql57-jdk8/4/6</a><pre>     [exec] * What went wrong:
     [exec] Execution failed for task ':runPoshi'.
     [exec] &gt; There were failing tests. See the report at: file:///opt/dev/projects/github/liferay-portal/portal-web/tests/index.html
     [exec] 
     [exec] * Try:
     [exec] Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.
     [exec] 
     [exec] * Get more help at https://help.gradle.org
     [exec] 
     [exec] BUILD FAILED in 4m 41s
     [exec] Result: 1
     [copy] Copying 1 file to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/394581
     [copy] Copying 2 files to /opt/dev/projects/github/liferay-portal/test-portal-acceptance-pullrequest-downstream(master)/builds/394581/screenshots
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1221541785.properties

clean-up-selenium-driver:
     [echo] Executing commands: 
     [echo] killall chromedriver
     [exec] chromedriver: no process found
     [exec] Result: 1
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1221541785.properties

clean-up-logs:
     [echo] stop.app.server.0.start.timestamp: 01-10-2023 20:45:49:813 PST
[beanshell] 2023-01-11 04:45:51.601 INFO  [main][ThemeHotDeployListener:171] 1 theme for speedwell-theme was unregistered
[beanshell] 2023-01-11 04:45:51.822 INFO  [main][ThemeHotDeployListener:171] 1 theme for admin-theme was unregistered
[beanshell] 2023-01-11 04:45:52.603 INFO  [main][ThemeHotDeployListener:171] 1 theme for dialect-theme was unregistered
[beanshell] 2023-01-11 04:45:52.778 INFO  [main][ThemeHotDeployListener:171] 1 theme for classic-theme was unregistered
[beanshell] 2023-01-11 04:45:53.374 INFO  [main][ThemeHotDeployListener:171] 1 theme for minium-theme was unregistered
[stopwatch] [stop.app.server.0: 16.022 sec]
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1221541785.properties

run-poshi-evaluate-console:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1125138336.properties

prepare-poshi-runner-properties:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/portal-web/poshi-ext.properties
[propertyfile] Updating property file: /opt/dev/projects/github/liferay-portal/.gradle/gradle.properties
     [echo] Executing Gradle task: evaluatePoshiConsole</pre></div></li><li><div><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369089//consoleText">functional-tomcat90-mysql57-jdk8/1/4</a><ol><li><div><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/369089//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_SimilarResults_ConfigureFields_">LocalFile.SimilarResults#ConfigureFields</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-01/test-1-37/test-portal-acceptance-pullrequest(master)/7873/functional-tomcat90-mysql57-jdk8/1/4/LocalFile.SimilarResults_ConfigureFields/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-01/test-1-37/test-portal-acceptance-pullrequest(master)/7873/functional-tomcat90-mysql57-jdk8/1/4/LocalFile.SimilarResults_ConfigureFields/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-01/test-1-37/test-portal-acceptance-pullrequest(master)/7873/functional-tomcat90-mysql57-jdk8/1/4/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376273//consoleText">functional-tomcat90-mysql57-jdk8/1/11</a><ol><li><div><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/376273//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_Blueprints_PaginateElementsAdmin_">LocalFile.Blueprints#PaginateElementsAdmin</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-01/test-1-37/test-portal-acceptance-pullrequest(master)/7873/functional-tomcat90-mysql57-jdk8/1/11/LocalFile.Blueprints_PaginateElementsAdmin/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-01/test-1-37/test-portal-acceptance-pullrequest(master)/7873/functional-tomcat90-mysql57-jdk8/1/11/LocalFile.Blueprints_PaginateElementsAdmin/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-01/test-1-37/test-portal-acceptance-pullrequest(master)/7873/functional-tomcat90-mysql57-jdk8/1/11/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li><div><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312499//consoleText">functional-tomcat90-mysql57-jdk8/1/12</a><ol><li><div><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/312499//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_Blueprints_PaginateBlueprintsPreview_">LocalFile.Blueprints#PaginateBlueprintsPreview</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-01/test-1-37/test-portal-acceptance-pullrequest(master)/7873/functional-tomcat90-mysql57-jdk8/1/12/LocalFile.Blueprints_PaginateBlueprintsPreview/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-01/test-1-37/test-portal-acceptance-pullrequest(master)/7873/functional-tomcat90-mysql57-jdk8/1/12/LocalFile.Blueprints_PaginateBlueprintsPreview/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-01/test-1-37/test-portal-acceptance-pullrequest(master)/7873/functional-tomcat90-mysql57-jdk8/1/12/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li><li>...</li></ol></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-37/jobs/test-portal-acceptance-pullrequest(master)/builds/7873/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-37/jobs/test-portal-acceptance-pullrequest(master)/builds/7873/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-37/jobs/test-portal-acceptance-pullrequest(master)/builds/7873/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest(master)/7873/">test-portal-acceptance-pullrequest(master)#7873</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-37.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/7873/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>search<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-search/liferay-portal/pull/728">liferay-search#728</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2156258850">[master] ci:test:search - thektan &gt; liferay-search - PR#728 - 2023-01-10[19:58:39]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-37.liferay.com/job/publish-testray-report/18102/">publish-testray-report#18102</a><br/></div></div>

thektan commented on modules/dxp/apps/search-experiences/search-experiences-web/src/main/resources/META-INF/resources/semantic_search/js/configuration/index.js:
```diff
@@ -57,166 +93,221 @@ const transformToLabelValueArray = (items = {}) => {
-		if (values.model === '') {
-			errors.model = Liferay.Language.get('this-field-is-required');
-		}
+	const initialTextEmbeddingProviderConfigurationsRef = useRef(
+		parseJSONString(initialTextEmbeddingProviderConfigurations)
+	);
```
If `semanticSearchCompanyConfigurationDisplayContext.getTextEmbeddingProviderConfigurations()` can return an array, we can cover both array and string cases by updating this line to:

```suggestion
	const initialTextEmbeddingProviderConfigurationsRef = useRef(
		Array.isArray(initialTextEmbeddingProviderConfigurations) ? initialTextEmbeddingProviderConfigurations : parseJSONString(initialTextEmbeddingProviderConfigurations)
	);
```

thektan commented:
@peerkar Pushed up the new changes, but I am running into an issue where when you save, you'll still see the old form data. But if you refresh the page, you'll see the updated data. I'm not sure why this happens at the moment if you had any ideas.

Changes:
- Removed `!_semanticSearchConfiguration.textEmbeddingsEnabled()`
- Removed `txtai` in property names
- Updated input names "Language IDs" -> "Languages" and "Asset Entry Class Names" -> "Asset Types"
- Fixed form submission data format. The hidden input needed to be the object itself (without the array brackets) `{...}` since there seems to be logic in the backend that is combining these `String[]` configurations into an array.

Remaining work on my end:
- Updating the test connection error message if the response is null

thektan commented:
@peerkar Test connection button should be fixed now. I also cleaned up my commits and squashed the fixes into my first frontend commit b85fe99f89fdf7d90fd596aec87cf04fab5e5aae.

I also removed my `gw buildLang` commit for the moment to avoid any additional rebase conflicts for the moment.

thektan commented:
@peerkar Force pushed an update to the format of the input, but the error still occurs where when you save, the default form will render. Feel free to forward this though and we can open a bug ticket that the US&M team could look into.

https://github.com/liferay-search/liferay-portal/pull/728/commits/42cfc9aff45cf585ece7b77912167d8a75ba870c#diff-7ff3b9ceb2c0a1fd670b04a2c8fec86d9dd98dbbc8eb3492b5732bdedf855fa0R914-R922

thektan commented:
New PR: https://github.com/liferay-search/liferay-portal/pull/735
